build/checks/spelling.py

```diff
@@ -12,11 +12,13 @@
 most = {}
 
 for artist in filter(lambda x: os.path.isdir(os.path.join(ROOT, x)), os.listdir(ROOT)):
+    if artist == "reginabellringers.ca": continue
     for dir in os.walk(os.path.join(ROOT, artist)):
         for file in dir[2]:
-            if not file.endswith(".html"): continue
+            if not file.endswith(".html") and not file.endswith(".htm"): continue
+            if os.path.join(ROOT, artist, file)=="./www/artists/ahasiw/sake.htm": continue
```

Comment:
What is the context for this page being excluded?

Reply:
Ahasiw Maskigon-Iskew's SAKEHI' TOWIN ONIPOWAK is a billingual poem written in both English and Cree. Including it would require adding dozens of cree words to the exclusion dictionary with the sole purpose of getting this page to pass. While all of the cree words are in full uppercase (which would help prevent accidental passes due to typoes), excluding the page seems like a better option. Here's a link to the page in the repository to see just what it is: https://github.com/SaskArtists/SaskArtists/blob/master/www/artists/ahasiw/sake.htm

Reply:
And here it is on SaskArtists:
https://saskartists.ca/artists/ahasiw/sake.htm